ui/src/App.tsx

```diff
@@ -90,10 +76,10 @@ function App() {
             </div>
             .{" "}
             <Row justify="center" gutter={[16, 42]}>
```

Comment:
Since you take out error property from useSearchLecturesQuery you should make use of it to prevent unexpected behaviour on error (it can be like this `        {error ? (
          <Card>Oh no, there was an error</Card>
        ) : isLoading ? (
          <LOADING_ICON />
        ) : data ? (
          data?.map....`

Reply:
Yes good point, added. 
I'm not a fan of putting that much logic in the render section so I put everything outside 👍